src/VirtoCommerce.ElasticSearchModule.Data/ElasticSearchRequestBuilder.cs

```diff
@@ -93,6 +98,14 @@ protected virtual ISort GetSortingField(SortingField field)
                     Order = geoSorting.IsDescending ? SortOrder.Descending : SortOrder.Ascending,
                 };
             }
+            else if (field.FieldName.ToLowerInvariant() == Score) 
```

Comment:
Could you use our extension `string.EqualsInvariant` to compare strings

Reply:
Fixed